### PR TITLE
Fix min/max properties of bool8 stype

### LINKED ISSUE #2231: stype.bool8.min/max are not boolean
`stype.bool8.min` returns 0, which is not a suitable value to be put into a boolean column. This prevents some generic code that creates a column from "min" value of each stype from working.

### PR DESCRIPTION
**Pull request by st-pasha**
(We may want to include this into 0.10.1 release as well)

Closes #2231

### CODE REVIEW

File: docs/changelog/v-0-11-0.rst [hunk 1]
```diff
@@ -9,5 +9,20 @@ Frame
...
 
 - |fix| A list of frames now displays properly in a Jupyter lab (#2222).
+
+
+
+General
+-------
+
+- |api| Properties ``dt.stype.min`` and ``dt.stype.max`` are now equal to
+  ``False`` and ``True`` respectively, instead of integers 0 and 1 (#2231).
```
Comment (oleksiyskononenko) [inline docs/changelog/v-0-11-0.rst]: Should we add "For boolean columns..."?

Comment (st-pasha) [reply docs/changelog/v-0-11-0.rst]: You right, it's a typo

Comment (st-pasha) [reply docs/changelog/v-0-11-0.rst]: *You're

what's up with the typos today? 😉

**Comment by st-pasha** [comment]
No point in waiting for PPC, going to merge

**Pull request merged**